wordpress-mobile/WordPress-iOS
# Switch About screen to use AutomatticAbout Swift package

Implements #17498. This PR removes the new About screen code from the main WordPress project and instead makes use of a Swift package that contains the same functionality.

There should be no functional changes here, the code for the About screen just now resides in a different location.

**To test**

* Build and run
* Navigate to Me > About and ensure that everything looks and works as expected.

## Regression Notes

1. Potential unintended areas of impact

None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
